libevmasm/KnownState.cpp

```diff
@@ -304,7 +304,7 @@ KnownState::StoreOperation KnownState::storeInStorage(
 
 	AssemblyItem item(Instruction::SSTORE, _location);
 	Id id = m_expressionClasses->find(item, {_slot, _value}, true, m_sequenceNumber);
-	StoreOperation operation(StoreOperation::Storage, _slot, m_sequenceNumber, id);
+	StoreOperation operation {StoreOperation::Storage, _slot, m_sequenceNumber, id};
```

Comment:
No space in front of `{`

Reply:
Actually, I personally prefer spacing here, for readability. Also goes in line with `inline class member initialization`, such as `int m_value {1234};`. it would just look unreadable in not having that space there (but that's my opinion). What do you think? Maybe we should globally revisit our coding style guidelines in general (as a team). :)